### ISSUE TITLE
(Spike): Investigate accessibility unit tests failing for Timeline component

### ISSUE THREAD

**Issue by flcbmajoka** [Royal-Navy/design-system#1446]
### Steps to reproduce
Performing accssibility test on Timeline component using `jest-axe`. Please see below steps to reproduce.

1. ![BookingTimeline](https://user-images.githubusercontent.com/66831714/93346858-55cd7880-f82c-11ea-99ea-7d6df0153550.jpeg)

2. ![BookingTimelineTest](https://user-images.githubusercontent.com/66831714/93346861-56660f00-f82c-11ea-9f3b-94b920400844.jpeg)

3. ![AccessibilityTestErrorMsg](https://user-images.githubusercontent.com/66831714/93346855-5534e200-f82c-11ea-988e-c0515f62bcc8.jpeg)

### Screenshot
*Add a screenshot*

**Comment by thyhjwb6** [comment]
This issue has been reproduced with the test:

```
  describe('axe', () => {
    it('should pass', async () => {
      expect.extend(toHaveNoViolations)

      const wrapper = render(
        <Timeline
          startDate={new Date(2020, 0, 1)}
          today={new Date(2020, 0, 15)}
        >
          <TimelineTodayMarker />
          <TimelineMonths />
          <TimelineWeeks />
          <TimelineDays />
          <TimelineRows>
            <TimelineRow name="Row 1">
              <TimelineEvents>
                <TimelineEvent
                  startDate={new Date(2020, 3, 14)}
                  endDate={new Date(2020, 3, 18)}
                >
                  Event 1
                </TimelineEvent>
              </TimelineEvents>
            </TimelineRow>
          </TimelineRows>
        </Timeline>
      )

      const results = await axe(wrapper.container)

      expect(results).toHaveNoViolations()
    })
  })
```

What it shows is that there is an edge case when `TimelineEvent` is not within range and the impact on accessibility. Because this is an edge case and the fix is complex, we are putting this on hold for now.

**Comment by thyhjwb6** [comment]
The fix for this is complex and would only affect an edge case.
